### DISCUSSION TITLE
Installation Error

### DISCUSSION THREAD

**Discussion by isaldwif** [pxlrbt/filament-excel#143] [Q&A]
Problem 1
    - pxlrbt/filament-excel[0.1.0, ..., 0.1.4, v1.0.0, ..., v1.1.13] require maatwebsite/excel ^3.1 -> found maatwebsite/excel[3.1.0, ..., 3.1.50] but it conflicts with your root composer.json require (^1.1).
    - pxlrbt/filament-excel[v2.0.0, ..., v2.1.8] require filament/filament ^3.0.0-stable -> found filament/filament[v3.0.0, ..., v3.0.94] but it conflicts with your root composer.json require (^2.0).
    - Root composer.json requires pxlrbt/filament-excel * -> satisfiable by pxlrbt/filament-excel[0.1.0, ..., 0.1.4, v1.0.0, ..., v1.1.13, v2.0.0, ..., v2.1.8]

**Comment by pxlrbt** [comment]
Please read the composer output:

> - pxlrbt/filament-excel[0.1.0, ..., 0.1.4, v1.0.0, ..., v1.1.13] require maatwebsite/excel ^3.1 -> found maatwebsite/excel[3.1.0, ..., 3.1.50] but it conflicts with your root composer.json require (^1.1).
